src/languages/armasm.js

```diff
@@ -7,6 +7,16 @@ Category: assembler
 
 export default function(hljs) {
     //local labels: %?[FB]?[AT]?\d{1,2}\w+
+
+  const COMMENT = {
+    variants: [
+      hljs.COMMENT('^[ ]*(?=#)', '$', {relevance: 0, excludeBegin: true }),
```

Comment:
Why just spaces, why not `\s`?

Reply:
`\s` eats new lines also and results in weirdness like the divs opening on the line before because the comment "includes" the newline.

Reply:
Ok. How about adding `\t` then?

Reply:
We could do that.  Are we good to go if we add support for tabs?

Reply:
Yep.